src/core/ext/lb_policy/round_robin/round_robin.c

```diff
@@ -102,12 +121,19 @@ typedef struct {
   ready_list *ready_list_node;
   /** last observed connectivity */
   grpc_connectivity_state connectivity_state;
+  /** the subchannel's target LB token */
+  grpc_lb_policy_address_token *lb_token;
 } subchannel_data;
 
 struct round_robin_lb_policy {
   /** base policy: must be first */
   grpc_lb_policy base;
 
+  /** total number of addresses received at creation time */
+  size_t num_addresses;
+  /** load balancing tokens, one per incoming address */
+  grpc_lb_policy_address_token *lb_tokens;
```

Comment:
It's not entirely clear to me why the round_robin LB policy needs to know about the LB token.  Shouldn't that be fully contained within the grpclb LB policy?

This may be something that we should discuss offline.